ios/Podfile.lock

```diff
@@ -16,30 +16,44 @@ PODS:
   - GoogleToolboxForMac/Defines (2.1.1)
   - GoogleToolboxForMac/Logger (2.1.1):
     - GoogleToolboxForMac/Defines (= 2.1.1)
-  - GoogleToolboxForMac/NSData+zlib (2.1.1):
+  - "GoogleToolboxForMac/NSData+zlib (2.1.1)":
     - GoogleToolboxForMac/Defines (= 2.1.1)
   - Protobuf (3.3.0)
-  - React (0.53.3):
-    - React/Core (= 0.53.3)
+  - React (0.55.4):
+    - React/Core (= 0.55.4)
   - react-native-background-timer (2.0.0):
     - React
-  - React/Core (0.53.3):
-    - yoga (= 0.53.3.React)
-  - yoga (0.53.3.React)
+  - React/Core (0.55.4):
+    - yoga (= 0.55.4.React)
+  - RNKeychain (3.0.0-rc.3):
+    - React
+  - yoga (0.55.4.React)
 
 DEPENDENCIES:
   - FirebaseMessaging
   - React (from `../node_modules/react-native`)
   - react-native-background-timer (from `../node_modules/react-native-background-timer`)
+  - RNKeychain (from `../node_modules/react-native-keychain`)
```

Comment:
Is this related to the PR?

Reply:
I get the same changes automatically in Xcode in a different PR https://github.com/status-im/status-react/pull/4988/files , probably at some point some code got through which did not update these files, so it's probably safer to include them at some point, otherwise we might be using different versions locally from prod (0.53 vs 0.55 seems like a big difference), but I know very little of mac build